packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -424,7 +424,10 @@ export class ProjectService extends BaseService {
             ) ||
             user.ability.cannot(
                 'create',
-                subject('Project', { organizationUuid }),
+                subject('Project', {
```

Comment:
I can't replicate this issue locally as an admin. :thinking: I don't think this type is needed, since it was only checking if you can `crate project in the org` , and an admin can `manage projects`